udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -299,7 +311,7 @@
   const warningRules = {
     acronym: {},
     business_number_id: { custom: checkBusinessId },
-    description: {required, minLengthValue: minLengthWarning(quality_description_length) },
+    description: {required, minLengthValue: minLengthWarning(parseInt(quality_description_length ?? "0")) },
```

Comment:
Shouldn't be better to have the `parseInt` in the config to always export `number | null` for `quality_description_length`?

Reply:
Yes ! Now that's it's a ts file, it's way better to type it in config